### ISSUE TITLE
Npm package is corrupted

### ISSUE THREAD

**Issue by webmarkyn** [nerdyman/react-image-turntable#30]
Hello @nerdyman. I was trying to install the new `3.1.0` version today and got the following error when trying to run the project `EvalError: Code generation from strings disallowed for this context`.  `dist/index.mjs` file looks strange and contains strings with absolute paths like
```
var e="/Users/me/Documents/projects/nerdyman/react-image-turntable/src/hooks.ts"
```

When I run `pnpm build` I get completely different output that's not even close to the `index.mjs` file that I get when installing a package from `npm`.

`NPM VERSION`
<img width="781" alt="Screenshot 2023-12-13 at 18 50 53" src="https://github.com/nerdyman/react-image-turntable/assets/23281442/e7e7ab8f-7d11-4206-951f-586374f75828">

`PNPM OUTPUT`
<img width="956" alt="Screenshot 2023-12-13 at 18 51 19" src="https://github.com/nerdyman/react-image-turntable/assets/23281442/76661f95-0ad5-40a4-9622-d5b9b294a481">

**Comment by nerdyman** [comment]
Ah that's really wierd, thanks for opening this @webmarkyn. I changed some of the build config stuff in the last release so I've probably misconfigured something there. I'll take a look ASAP.

**Comment by nerdyman** [comment]
It's the `np` lib used to generate releases doing this, during the "Git" or "Bumping version using npm" step it seems to run the tests even though it's configured to skip them. The tests use a different build to generate coverage so that's why the build output is borked. I'm a bit fed up with np and was already going to replace #26 so will do that and get a _working_ build out today.

**Comment by webmarkyn** [comment]
Hello @nerdyman, have you managed to resolve it?

**Comment by nerdyman** [comment]
Sorry for the delay @webmarkyn, this has been fixed in `3.1.1`. I've tested it in a demo project and everything looks OK on unpkg too.

https://unpkg.com/browse/react-image-turntable@3.1.1/dist/index.mjs
